src/Tizen.Multimedia/Common/IWindowProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// The IWindowProvider interface provides the window handle and information about the window's position, size, and rotation.
+    /// </summary>
+    /// <since_tizen> 12 </since_tizen>
+    public interface IWindowProvider
+    {
+        /// <summary>
+        /// Gets the window handle
+        /// </summary>
+        /// <remarks>
+        /// This handle represents Ecore_Wl2_Window on native.
+        /// </remarks>
+        IntPtr WindowHandle { get; }
+
+        /// <summary>
+        /// Gets the x-coordinate of the window's position.
+        /// </summary>
+        float X { get; }
+        /// <summary>
+        /// Gets the y-coordinate of the window's position.
+        /// </summary>
+        float Y { get; }
+
+        /// <summary>
+        /// Gets the width of the window.
+        /// </summary>
+        float Width { get; }
+
+        /// <summary>
+        /// Gets the height of the window.
+        /// </summary>
+        float Height { get; }
+
+        /// <summary>
+        /// Gets the rotation of the window in degrees. The value can only be 0, 90, 180, or 270.
```

Comment:
I was going to ask for more description to show which value correspond to which orientation, but I guess this whole interface is more for intra module interfacing rather than to provide information to applications. So OK.